pkg/testing/integration_test.go

```diff
@@ -75,6 +78,13 @@ func newSecretProfile() *secretProfile {
 	}
 }
 
+var (
+	setUpOnce = sync.Once{}
+	setUpDone = make(chan struct{})
+)
+
+const controllerNamespace = "integration-test-controller"
+
 func (s *IntegrationSuite) SetUpSuite(c *C) {
 	ctx := context.Background()
 	ctx, s.cancel = context.WithCancel(ctx)
```

Comment:
@tdmanv I think you've got to manage the cancel for the controller outside of the Suite construct.

More I think about it the cleaner it is to have a setup and teardown func in this file that is called by the Test func in `testing_test.go`.